drivers/hud_apr/app/models/hud_apr/generators/shared/fy2021/question_ten.rb

```diff
@@ -105,12 +105,15 @@ def self.table_descriptions
       ]
       columns = {
         'B' => Arel.sql('1=1'),
-        'C' => a_t[:age].between(0..17).and(a_t[:dob_quality].in([1, 2])),
-        'D' => a_t[:age].between(18..24).and(a_t[:dob_quality].in([1, 2])),
-        'E' => a_t[:age].between(25..61).and(a_t[:dob_quality].in([1, 2])),
-        'F' => a_t[:age].gteq(62).and(a_t[:dob_quality].in([1, 2])),
+        'C' => a_t[:age].between(0..17).and(a_t[:dob_quality].in([1, 2, 99])),
+        'D' => a_t[:age].between(18..24).and(a_t[:dob_quality].in([1, 2, 99])),
+        'E' => a_t[:age].between(25..61).and(a_t[:dob_quality].in([1, 2, 99])),
+        'F' => a_t[:age].gteq(62).and(a_t[:dob_quality].in([1, 2, 99])),
```

Comment:
do we need the dob_quality check at all?  That looks like all possible options right?